src/main/webapp/i18n/de/assessment.json

```diff
@@ -42,7 +42,8 @@
                 "saveSuccessful": "Deine Bewertung wurde erfolgreich gespeichert!",
                 "saveFailed": "Das Speichern deiner Bewertung ist fehlgeschlagen!",
                 "updateAfterComplaintSuccessful": "Die Bewertung wurde erfolgreich aktualisiert.",
-                "updateAfterComplaintFailed": "Das Aktualisieren der Bewertung ist fehlgeschlagen."
+                "updateAfterComplaintFailed": "Das Aktualisieren der Bewertung ist fehlgeschlagen.",
+                "removeGradingInstructionLink": "Möchten Sie den Link zur Benotungshinweise entfernen?"
```

Comment:
confused again. grading criterion is wrapper of grading instructions. So, feedback is related with grading instruction not directly with grading criterion. So, which one is right for grading instruction? Benotungshinweise or Benotungskriterium 😄 Benotungshinweise is suggested for grading instruction, while i was working on my other PR. I am just tryin to learn...

Reply:
I see, I mixed that up, sorry. I wasn't fully aware these are two different things. In that case it should be 
```suggestion
                "removeGradingInstructionLink": "Möchten Sie den Link zum Benotungshinweis entfernen?"
```
(otherwise it would be plural)

Reply:
thank youuuu, i will fix it 😊